lib/basedriver/driver.js

```diff
@@ -470,10 +470,11 @@ class BaseDriver extends Protocol {
    * should be made available to overridden methods.
    * @param {string} method - HTTP method of the route
    * @param {string} url - url of the route
+   * @param {*?} body - webdriver request body
```

Comment:
`{*?}` -> `{?*}`

Reply:
oh, i thought optionality was designated after the type, probably thinking of some other language like typescript maybe.